dockerfiles/transaction-metrics-exporter/Dockerfile

```diff
@@ -18,13 +18,19 @@ COPY scripts/ scripts/
 # Copy only pkg.json
 COPY packages/typescript/package.json packages/typescript/
 COPY packages/utils/package.json packages/utils/
+COPY packages/protocol/package.json packages/protocol/
```

Comment:
@Pedro-vk we probably want to point here to latest contractkit?

Reply:
or maybe after merge changes we made here to contractKit and released a new version?

Reply:
I think that point to a specific version to avoid retrocompatibility issues

Reply:
Yeah we'll want to use the published version as much as possible to avoid bloating the docker image here